packages/api-db-mongodb/src/db/schema.ts

```diff
@@ -100,11 +101,14 @@ export interface DBUser {
   modifiedAt: Date
 
   email: string
+  emailVerified: Date | null
```

Comment:
Would it be better to have an indication in the naming that this is a date? `emailVerifiedAt` or something?